src/IntrType.f90

```diff
@@ -1000,6 +1008,19 @@ ELEMENTAL FUNCTION notequalto_logical(l1,l2) RESULT(bool)
     ENDFUNCTION notequalto_logical
 !
```

Comment:
xor_logical is the same function as notequalto_logical.

Reply:
Ah, good call... totally didn't know this was in there.  Do we want to revert the change, or just leave both?